### PR TITLE
Allow customization of the default layout name in mode-line

### PR DESCRIPTION
**Pull request by nixmaniack**
"Default" wasn't looking good! :wink:

### CODE REVIEW

**Comment by syl20bnr** [comment]
I first try it but it require more coding to be able to hide it so I deferred it. I must have missed a simple way to do it so feel free to try :-)

**Comment by nixmaniack** [comment]
This PR just allows to configure the name as you want using the `dotspacemacs-display-default-layout "@Home"` keeping everything as it is.

**Comment by syl20bnr** [comment]
Oh this is a PR :-)
Octocat client confused me, issues and PRs are in the same list.

**Comment by nixmaniack** [comment]
Hah! Your reply was confusing to me hence I commented with the word "PR" in it!

**Comment by syl20bnr** [comment]
Sorry for the confusion.

File: core/core-dotspacemacs.el [hunk 1]
```diff
@@ -121,7 +121,7 @@ with `:' and Emacs commands are executed with `<leader> :'.")
   "If non nil `Y' is remapped to `y$'.")
 
 (defvar dotspacemacs-display-default-layout nil
-  "If non nil then the `Default' layout name is displayed in the mode-line.")
+  "Name of the default layout as a string to be displayed in the mode-line.")
```
Comment (syl20bnr) [inline core/core-dotspacemacs.el]: You can rename the variable to reflect that this is about the name of the default layout and update the documentation to mention that a nil name will not show up in the mode-line.

Comment (nixmaniack) [inline core/core-dotspacemacs.el]: Updated!

**Comment by TheBB** [comment]
Thanks! Cherry-picked in develop. You can safely delete your branch.

**Pull request closed without merging**